modular_skyrat/code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -0,0 +1,225 @@
+GLOBAL_LIST_INIT(maintenance_loot, list(
+	//Random stuff first
+	/obj/item/poster/random_contraband = 50,
+	/obj/item/poster/random_official = 25,
+	/obj/item/book/random/triple = 5,
+	/obj/item/clothing/head/wig/random = 5,
+	/obj/item/clothing/under/color/jumpskirt/random = 5,
+	/obj/item/clothing/under/color/random = 10,
+	/obj/item/dildo/random = 5,
+	/obj/item/lipstick/random = 3,
+	/obj/item/organ/random = 10, //I mean yeah it would decay but it would be good fluff.
+	/obj/item/reagent_containers/blood/random = 10,
+	/obj/item/reagent_containers/food/snacks/grown/random = 5,
+	/obj/item/reagent_containers/rag/towel/random = 10,
+	/obj/item/restraints/handcuffs/cable/random = 5,
+	/obj/item/seeds/random = 1,
+	/obj/item/stack/cable_coil/random = 100,
+	/obj/item/storage/wallet/random = 10,
+	/obj/item/tape/random = 5,
+	/obj/item/toy/plush/random = 5,
+	/obj/item/toy/plush/random_snowflake = 1,
+	/obj/item/paint/anycolor = 25,
+
+	//Loot here.
+	/obj/effect/spawner/lootdrop/breathing_masks = 200,
+	/obj/effect/spawner/lootdrop/breathing_tanks = 50,
+	/obj/effect/spawner/lootdrop/cig_packs = 50,
+	/obj/effect/spawner/lootdrop/cigars_cases = 10,
+	/obj/effect/spawner/lootdrop/coin = 50,
+	/obj/effect/spawner/lootdrop/costume = 25,
+	/obj/effect/spawner/lootdrop/druggie_pill = 10,
+	/obj/effect/spawner/lootdrop/gloves = 100,
+	/obj/effect/spawner/lootdrop/glowstick = 200,
+	/obj/effect/spawner/lootdrop/healing_kits = 25,
+	/obj/effect/spawner/lootdrop/low_tools = 500,
+	/obj/effect/spawner/lootdrop/mre = 100,
+	/obj/effect/spawner/lootdrop/organ_spawner = 10,
+	/obj/effect/spawner/lootdrop/space_cash = 100,
+	/obj/effect/spawner/lootdrop/tool_box = 200,
+	/obj/effect/spawner/lootdrop/welder_tools = 50,
+
+	//Other here.
+	/obj/item/a_gift = 50,
```

Comment:
This can give Boh and null rods, maybe not the best idea

Reply:
I'll remove then.